build-system/tasks/helpers.js

```diff
@@ -542,6 +542,22 @@ async function minify(code, map) {
         // eslint-disable-next-line local/camelcase
         keep_quoted: /** @type {'strict'} */ ('strict'),
       },
+      // eslint-disable-next-line local/camelcase
+      nth_identifier: {
+        get(num) {
+          const leading =
+            '$ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz';
+          const all = leading + '0123456789';
+          let charset = leading;
+          let id = '';
+          do {
+            id += charset[num % charset.length];
```

Comment:
The first sequence of chars goes as I'd expect. The second round I found surprising:
1. A
2. B
3. C
4. ...

55: 'AA'
56: 'BA'
57: 'CA'

Is that intentional? I would have expected AA, AB, AC

Reply:
It's the easiest way to handle it, but I don't believe it makes a difference.

Reply:
In that case, LGTM